things/api/things/http/endpoint_test.go

```diff
@@ -126,7 +127,7 @@ func TestCreateThing(t *testing.T) {
 			contentType: contentType,
 			auth:        token,
 			status:      http.StatusCreated,
-			location:    "/things/001",
+			location:    "/things/123e4567-e89b-12d3-a456-000000000001",
```

Comment:
From where this location comes? Please, try to use consts as much as posible to make it understandable.

Reply:
This location is also finally coming from pkg/uuid/mock.go only.  This Mock UUID is assigned during Thing / Channel creation.  More over the the same ID is used for storing inside things/mock directory also.  

We can see similar location used in twins/api/http/endpoint_twins_test.go:126 etc.

Reply:
Same here, simply use the const from the package. It's self explanatory.